packages/ui-components/style/base.css

```
@@ -51,12 +51,13 @@ strong {
   font-weight: unset;
 }
 
-/* Override Blueprint's _typography.scss styles */
-a {
+/* Override Blueprint's _typography.scss styles outside of dialog boxes */
```

There is no Blueprint anymore, this is outdated

```suggestion
/* Disable native link decoration styles everywhere outside of dialog boxes */
```